tests/src/test/java/alluxio/server/ft/FileSystemMasterMergeJournalContextIntegrationTest.java

```diff
@@ -0,0 +1,144 @@
+/*
```

Comment:
FileSystemMasterMergeJournalContextIntegrationTest @tcrain this test to test the InodeTree.getDescendants case

Reply:
Can you make a simple unit test that makes sure the flush is being called correctly when this is set PropertyKey.MASTER_RECURSIVE_OPERATION_JOURNAL_FORCE_FLUSH_MAX_ENTRIES. For example create a mock that counts the number of flushes and size of journal when it is called during a recursive delete operation please?

Reply:
sure sure